### PR TITLE
feat(specs): source update changes

### PR DESCRIPTION
**Pull request by mehmetaligok**
## 🧭 What and Why

🎟 JIRA Ticket: https://algolia.atlassian.net/browse/DI-827

### Changes included:

* SourceUpdate endpoint changed to put
* SourceUpdate input ref fixed

### CODE REVIEW

File: specs/ingestion/common/schemas/source.yml [hunk 1]
```diff
@@ -150,3 +148,36 @@ SourceInput:
     - $ref: '#/SourceCommercetools'
     - $ref: '#/SourceBigCommerce'
     - $ref: '#/SourceJson'
+
+SourceUpdateCommercetools:
+  type: object
+  additionalProperties: false
+  properties:
+    storeKeys:
+      type: array
```
Comment (shortcuts) [inline specs/ingestion/common/schemas/source.yml:157]:
```suggestion
      type: array
      description: Unique and immutable key of the referenced Store
```

description comes from the CT docs directly

File: specs/ingestion/common/schemas/source.yml [hunk 2]
```diff
@@ -150,3 +148,36 @@ SourceInput:
...
+        Array of locales that must match the following pattern: ^[a-z]{2}(-[A-Z]{2})?$. For example ["fr-FR", "en"].
+      items:
+        type: string
+
+SourceUpdateJson:
+  type: object
+  additionalProperties: false
+  properties:
+    url:
+      type: string
```
Comment (shortcuts) [inline specs/ingestion/common/schemas/source.yml:172]:
```suggestion
      type: string
      description: The URL of the file.
```

**Review by shortcuts — COMMENTED** [review]
looks great :)

File: specs/ingestion/common/schemas/source.yml (hunk 1 above)
Comment (mehmetaligok) [reply specs/ingestion/common/schemas/source.yml:160]: added and also original code part is updated with descriptions.

**Review by shortcuts — APPROVED** [review]
:D

**Pull request merged**